packages/financial-templates-lib/src/clients/OptimisticOracleClient.ts

```diff
@@ -212,36 +261,28 @@ export class OptimisticOracleClient {
     ).filter(isDefined);
 
     // Filter out disputes that were already settled and reformat data.
-    const unsettledResolvedDisputeEvents = (
```

Comment:
Same reasoning as [this comment](https://github.com/UMAprotocol/protocol/pull/3554/files#r742089288)